src/components/top-wrecker-card/ASTopWreckerCard.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react'
+import { Image, Pressable, Text } from 'react-native'
+
+import compassion from '../../assets/icons/compassion.png'
+import courage from '../../assets/icons/courage.png'
+import curiosity from '../../assets/icons/curiosity.png'
+import distractor from '../../assets/icons/distractor.png'
+import overthinker from '../../assets/icons/overthinker-grey.png'
+import procrastinator from '../../assets/icons/procrastinator.png'
+import ASLoadingBar from '../loading-bar/ASLoadingBar'
+import ASModal from '../modal/ASModal'
+
+import { Typography } from '../../theme/typography'
+
+import { styles } from './asTopWreckerCard-styles'
+
+interface IASTopWreckerCardProps {
+  image: string
+  percent: string
+  title: string
+  description: string
+}
+
+const ASTopWreckerCard = ({ image, percent, title, description }: IASTopWreckerCardProps) => {
+  const [showModal, setShowModal] = useState(false)
+
+  const wreckerImage: any = {
+    Procrastinator: procrastinator,
+    Overthinker: overthinker,
+    Courage: courage,
+    Compassion: compassion,
+    Curiosity: curiosity,
+    Distractor: distractor,
+  }
```

Comment:
move. in constant file 
use enum or if not using enum then specify type not any
const wreckerImage : {[key :string] : string} = {
.......
}